index.html

```diff
@@ -31,7 +31,7 @@
   <ul>
   {% assign future_meetings_planned = false %}
   {% for meeting in meetings %}
-    {% assign meeting_date = meeting.date | date: '%F' %}
+    {% assign meeting_date = meeting.date | date: '%F 18:30 PM' %}
```

Comment:
As above, I think this is only used for the conditional, perhaps it's unnecessary?

Reply:
Removed